tests/safeds/data/tabular/containers/_cell/test_first_not_none.py

```diff
@@ -0,0 +1,54 @@
+from datetime import date, time
+
+import pytest
+from safeds.data.tabular.containers._cell import Cell
+from safeds.data.tabular.containers._lazy_cell import _LazyCell
+
+
+class TestFirstNotNone:
+    def test_should_return_none(self) -> None:
+        to_eval: list[Cell] = [_LazyCell(None) for i in range(5)]
+        res = Cell.first_not_none(to_eval)
+        assert res.eq(_LazyCell(None))
+
+    @pytest.mark.parametrize(
+        ("list_of_cells", "expected"),
+        [
+            ([_LazyCell(None), _LazyCell(1), _LazyCell(None), _LazyCell(4)], _LazyCell(1)),
+            ([_LazyCell(i) for i in range(5)], _LazyCell(1)),
+            (
+                [_LazyCell(None), _LazyCell(None), _LazyCell("Hello, World!"), _LazyCell("Not returned")],
+                _LazyCell("Hello, World!"),
+            ),
+            ([_LazyCell(i) for i in ["a", "b", "c", "d"]], _LazyCell("a")),
```

Comment:
Should use [`pl.lit`](https://docs.pola.rs/api/python/stable/reference/expressions/api/polars.lit.html) to wrap the string literal. Otherwise, the string is interpreted as a column name, which is probably not wanted here.